flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -583,7 +588,7 @@ public void testPriorityQueueStateCreationFailsIfNewSerializerIsNotCompatible()
     //  Tests for key serializer in keyed state backends
     // -------------------------------------------------------------------------------
 
-    @Test
+    @TestTemplate
     public void testStateBackendRestoreFailsIfNewKeySerializerRequiresMigration() throws Exception {
```

Comment:
```suggestion
    void testStateBackendRestoreFailsIfNewKeySerializerRequiresMigration() throws Exception {
```
could be package private